lib/luaxtable.h

```diff
@@ -6,9 +6,14 @@
 #ifndef luaxtable_h
 #define luaxtable_h
 
+#define LUAXTABLE_USERDATA_SIZE 256
+
 struct luaxtable_s;
 
 typedef struct luaxtable_info_s {
+	char userdata[LUAXTABLE_USERDATA_SIZE];
```

Comment:
I would rename this.. `info`, `userinfo`, `arg`.. not sure what's the best option..